enterprise/frontend/src/metabase-enterprise/tools/index.js

```diff
@@ -24,6 +23,5 @@ if (hasPremiumFeature("audit_app")) {
       title={t`Erroring Questions`}
       component={ErrorOverview}
     />,
-    <Route key="error-detail" path="errors/:cardId" component={ErrorDetail} />,
```

Comment:
This component was used when we click into each errored question in Admin > Tools when running EE build
![image](https://github.com/metabase/metabase/assets/1937582/8e0e9bd1-4a19-4ad8-93d9-d42e4590c04e)

But the ability to click into the question detail was already removed in https://github.com/metabase/metabase/pull/39639

Going directly to `/admin/tools/errors/:cardId` doesn't work either. That's why this path and the underlying component should be removed.